system/lib/wasmfs/thread_utils.h

```diff
@@ -25,9 +25,7 @@ class ProxyWorker {
   // Spawn the worker thread.
   ProxyWorker()
     : queue(), thread([&]() {
-        // Process the queue in case any work came in while we were starting up,
-        // then sit in the event loop performing work as it comes in.
-        queue.execute();
+        // Sit in the event loop performing work as it comes in.
         emscripten_exit_with_live_runtime();
```

Comment:
Can we assert on something like `queue.empty()` here?

Though, I guess the queue must be empty since we just constructed it here in the constructor. But that previous PR didn't make that change AFAICT, so this was the case before...? Is there some other possible source of events somehow?

Reply:
No, the queue might not be empty here because the proxying thread may have already proxied some work by the time the worker thread gets here. But if the queue is not empty, then there will be a pending postMessage notification that will cause the worker thread to execute the queue immediately after this anyway.

Reply:
I see, thanks.